fix: (storage) update doc  to link google_storage_bucket_object_content in google_storage_bucket_object (#12854)
GoogleCloudPlatform/magic-modules: mmv1/third_party/terraform/website/docs/r/storage_bucket_object.html.markdown

and 
[API](https://cloud.google.com/storage/docs/json_api/v1/objects).

> A datasource can be used to retrive data of the object

Reviewer: ```suggestion
A datasource can be used to retrieve the data of the stored object:
```

> * `google_storage_bucket_object_content`: Retrieves the data of the object

Reviewer 2: Can you please add link to the data source? It will help users to directly navigate to data source reference documentation.
nit: "Retrieves the data of the object."

Author: the above format will finally link the page URL to the data source when published in documentation

updated description

Author: added above like this 

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/storage_managed_folder_iam.html.markdown#iam-policy-for-cloud-storage-managedfolder

Reviewer 2: Sounds Good!

Author: yes release-note is none type

> * `google_storage_bucket_object_content`: Gets an existing object content inside an bucket in Google Cloud Storage service (GCS)

Reviewer: ```suggestion
* `google_storage_bucket_object_content`: Retrieves the content within a specified bucket object in Google Cloud Storage Service (GCS)
```
